assets/js/modules/analytics/datastore/accounts.test.js

```diff
@@ -57,10 +73,83 @@ describe( 'modules/analytics accounts', () => {
 	afterEach( () => {
 		unsubscribeFromAll( registry );
 		apiFetchSpy.mockRestore();
+		windowLocationSpy.mockRestore();
 	} );
 
 	describe( 'actions', () => {
+		describe( 'createAccount', () => {
+			it( 'creates an account ticket and redirects to the Terms of Service', async () => {
+				fetch
+					.doMockIf(
+						/^\/google-site-kit\/v1\/modules\/analytics\/data\/create-account-ticket/
+					)
+					.mockResponse(
+						JSON.stringify( fixtures.accountTicket ),
+						{ status: 200 }
+					);
+
+				muteConsole( 'error' );
+
+				registry.dispatch( STORE_NAME ).createAccount( { accountName, propertyName, profileName, timezone } );
+				await subscribeUntil( registry,
+					() => (
+						registry.select( STORE_NAME ).isDoingCreateAccount() !== true
```

Comment:
Nitpick, but I think it's easier to read this way:
```suggestion
						registry.select( STORE_NAME ).isDoingCreateAccount() === false
```

Reply:
Good suggestion. Possible because before the correct setup  (and !! casting) you suggested I wasn't certain this would `=== false`. I am now though and changed in https://github.com/google/site-kit-wp/pull/1390/commits/28571eef3c39f85248bd972864b2d1c129ab8db4.